### PR TITLE
Add ItemReview component

### PR DESCRIPTION
**Pull request by ds300**
[Jira](https://artsyproduct.atlassian.net/browse/PURCHASE-309)
[Zeplin](https://zpl.io/bzWx4KE)

This component is currently only used on desktop.

image

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 99042fec1eb83c0ec1d22d8d7177de3e28832313

https://deploy-preview-1128--artsy-reaction.netlify.com

File: src/Apps/Order/Components/__stories__/ItemReview.story.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            title="The Fox and the Hound"
+            date="2018"
+            medium="Oil on canvas"
+            dimensions={{
+              in: "96 × 79 in",
+              cm: "243.8 × 200.7 cm",
+            }}
+            attributionClassDescription="This is a unique work"
+            imageURL="https://d32dm0rphc51dk.cloudfront.net/XIlup0jb5BESj71JI7ZHpQ/larger.jpg"
+          />
```
Comment (damassi) [inline src/Apps/Order/Components/__stories__/ItemReview.story.tsx]: @ds300 - one thing we continually noticed while building out the Artist page from static comps was that defining prop names ahead of time always had to be unwound once the Metaphysics / Relay side of things was introduced. (If you're certain that these are the fields that will eventually flow in all good tho 👍)

**Comment by damassi** [comment]
LGTM 👍

File: src/Apps/Order/Components/__stories__/ItemReview.story.tsx (hunk 1 above)
Comment (damassi) [reply src/Apps/Order/Components/__stories__/ItemReview.story.tsx]: You can if you'd like (that's what we did while building out the new `/artwork page), but typically its better to leave the props-defining / wiring phase to a later stage when the component is actually being wired up into a view (meaning, wait until the component is plugged into relay). By keeping things totally static it increases flexibility during the design / comp phase.

**Pull request merged**